### ISSUE TITLE
Please update to Angular4

### ISSUE THREAD

**Issue by BjornMelgaard** [code-chunks/angular2-logger#165]
```
$ npm install --save angular2-logger
greenbadger-client@0.0.0 /home/bjorn/projects/greenbadger-client
├── UNMET PEER DEPENDENCY @angular/common@4.1.1
├── UNMET PEER DEPENDENCY @angular/compiler@4.1.1
├── UNMET PEER DEPENDENCY @angular/core@4.1.1
├── UNMET PEER DEPENDENCY @angular/platform-browser@4.1.1
├── UNMET PEER DEPENDENCY @angular/platform-browser-dynamic@4.1.1
└── angular2-logger@0.5.1 

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN ng2-charts@1.5.0 requires a peer of @angular/common@^2.3.0 but none was installed.
npm WARN ng2-charts@1.5.0 requires a peer of @angular/core@^2.3.0 but none was installed.
npm WARN angular2-logger@0.5.1 requires a peer of @angular/common@^2.0.0 but none was installed.
npm WARN angular2-logger@0.5.1 requires a peer of @angular/compiler@^2.0.0 but none was installed.
npm WARN angular2-logger@0.5.1 requires a peer of @angular/core@^2.0.0 but none was installed.
npm WARN angular2-logger@0.5.1 requires a peer of @angular/platform-browser@^2.0.0 but none was installed.
npm WARN angular2-logger@0.5.1 requires a peer of @angular/platform-browser-dynamic@^2.0.0 but none was installed.
```

**Comment by kichano** [comment]
I have the same problem.

**Comment by langley-agm** [comment]
Fixed in 0.6.0 let me know if any issues !
